src/clint.py

```diff
@@ -2305,213 +2304,36 @@ def CheckBraces(filename, clean_lines, linenum, error):
-                    r'(?:\(\d+(, \d+)*\))?',
-                    clean_lines.lines[func_start_linenum],
-                )
-                if attrline:
-                    if len(attrline.group(1)) != 2:
-                        error(filename, func_start_linenum,
-                              'whitespace/indent', 5,
-                              'Function attribute line should have 2-space '
-                              'indent')
-
-                    func_start_linenum += 1
-                else:
-                    func_start = clean_lines.lines[func_start_linenum]
-                    if not func_start.startswith('enum ') and func_start.endswith('{'):
-                        error(filename, func_start_linenum,
-                              'readability/braces', 5,
-                              'Brace starting function body must be placed '
-                              'after the function signature')
-                    break
-
-    # An else clause should be on the same line as the preceding closing brace.
-    # If there is no preceding closing brace, there should be one.
-    if Match(r'\s*else\s*', line):
-        prevline = GetPreviousNonBlankLine(clean_lines, linenum)[0]
-        if Match(r'\s*}\s*$', prevline):
-            error(filename, linenum, 'whitespace/newline', 4,
-                  'An else should appear on the same line as the preceding }')
-        else:
-            error(filename, linenum, 'readability/braces', 5,
-                  'An else should always have braces before it')
-
-    # If should always have a brace
-    for blockstart in ('if', 'while', 'for'):
-        if Match(r'\s*{0}(?!\w)[^{{]*$'.format(blockstart), line):
-            pos = line.find(blockstart)
-            pos = line.find('(', pos)
-            if pos > 0:
-                (endline, _, endpos) = CloseExpression(
-                    clean_lines, linenum, pos)
-                if endline[endpos:].find('{') == -1:
-                    error(filename, linenum, 'readability/braces', 5,
-                          '{} should always use braces'.format(blockstart))
-
-    # If braces come on one side of an else, they should be on both.
-    # However, we have to worry about "else if" that spans multiple lines!
-    if Search(r'}\s*else[^{]*$', line) or Match(r'[^}]*else\s*{', line):
-        if Search(r'}\s*else if([^{]*)$', line):       # could be multi-line if
-            # find the ( after the if
-            pos = line.find('else if')
-            pos = line.find('(', pos)
-            if pos > 0:
-                (endline, _, endpos) = CloseExpression(
-                    clean_lines, linenum, pos)
-                # must be brace after if
-                if endline[endpos:].find('{') == -1:
-                    error(filename, linenum, 'readability/braces', 5,
-                          'If an else has a brace on one side,'
-                          ' it should have it on both')
```

Comment:
is uncrustify catching this ?

Reply:
Lmao yes, I only removed things uncrustify already takes care of.